src/wiki/getting-started/installing-java.md

```diff
@@ -6,7 +6,17 @@ eleventyNavigation:
 ---
 # Installing Java
 
-Java is **required** in order to run Minecraft. As such, until Java downloading is implemented, you **will** need to **manually** install Java to run Minecraft with Prism Launcher.
+## Automatically installing Java (recommended)
+
+In Prism Launcher 9.0+, you can let the launcher handle Java automatically on Windows, macOS and Linux (not available in all packages) by enabling "Autodetect Java version" and "Auto-download Mojang Java" in:
+
+> Prism's main screen > Settings (top toolbar) > Java
```

Comment:
```suggestion
> Settings (top toolbar) > Java
```
Nit, but this is more in-line with other navigation instructions we have (as we can pretty safely assume most people will be on the main screen)